comp/trace/status/statusimpl/status_templates/traceagent.tmpl

```diff
@@ -0,0 +1,46 @@
+{{- with .apmStats }}
```

Comment:
@GustavoCaso Does this `comp/trace/status` supersede the `pkg/trace/status` package? If yes, is it required to maintain the `pkg` one?

Reply:
Since I'm migrating one `status` command at a time, I need to keep the other templates for now. So, the existing templates at `pkg/status/render/templates/*` can not be deleted yet, as I want the other status command to work in isolation from the `agent status` command. I will remove those once I migrate all `status` commands. 

I update the PR description so others reviewing the PR don't have the same question.